pkg/hcp/telemetry/exporter.go

```diff
@@ -0,0 +1,70 @@
+package telemetry
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/consul/proto-public/pbresource"
+	"github.com/hashicorp/go-hclog"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+)
+
+const exporterDefaultScrapeInterval = time.Minute
```

Comment:
would it be useful to make this configurable?

Reply:
Ever? Definitely. We had a plan for something like that from Consul servers -- CTGW would tell servers how frequently to push -- but in practice we think 1m is a pretty good default (none of our dashboards/graphs use anything at a lower resolution than that right now). If we supported it already in CTGW I'd be more inclined to support it here